<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayslipWidget</class>
 <widget class="QWidget" name="PayslipWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Payslip Generation Module</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>552</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="scrollAreaLayout">
       <item>
        <widget class="QLabel" name="labelPayrollPeriod">
         <property name="text">
          <string>Payroll Period:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboPayrollPeriod"/>
       </item>
       <item>
        <widget class="QLabel" name="labelEmployeeName">
         <property name="text">
          <string>Employee Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="employeeComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="labelPayslipImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="pixmap">
          <pixmap>:/resources/docs/images/payslip.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelOutputFormat">
         <property name="text">
          <string>Output Format:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="outputFormatLayout">
         <item>
          <widget class="QCheckBox" name="checkPDF">
           <property name="text">
            <string>PDF</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkExcel">
           <property name="text">
            <string>Excel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="actionButtonsLayout">
         <item>
          <widget class="QPushButton" name="generateAllButton">
           <property name="text">
            <string>Generate All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="generateSelectedButton">
           <property name="text">
            <string>Generate Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="printButton">
           <property name="text">
            <string>Print</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="emailButton">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
